Makefile

```diff
@@ -82,8 +82,15 @@ docgen: ## Generate docs
 	go run hack/docs/metrics_gen_docs.go pkg/ website/content/en/preview/tasks/metrics.md
 	go run hack/docs/instancetypes_gen_docs.go website/content/en/preview/AWS/instance-types.md
 	go run hack/docs/configuration_gen_docs.go website/content/en/preview/tasks/configuration.md
-     
-release: ## Generate release manifests and publish a versioned container image.
+
+release-gen: docgen ## Generate any materials which should be updated prior to release
```

Comment:
We generated new source code, the next step in release is to start building images.  We want to lint at a minimum and it doesn't hurt to run tidy and cache any new modules. I doubt the pricing code will ever import any new module, but there will potentially be future code generation that's performed at release time.

Reply:
What if you just added verify as a target dependency after release-gen? 

```
release: release-gen verify
...
```

If anything has changed, then it would fail, but I think that's okay, the generated code just needs to be committed and then rerun. Probably best to have that looked at someone separately anyways to ensure nothing looks wrong.

Reply:
https://www.gnu.org/software/make/manual/html_node/Prerequisite-Types.html